tck-flow/README.md

```diff
@@ -0,0 +1,623 @@
+# Reactive Streams TCK #
```

Comment:
Perhaps: "Reactive Streams TCK for java.util.concurrent.Flow"?

Reply:
Right, working on this today; should be done in one hour :)

Reply:
Like the name proposal btw, using that 👍